packages/ember-runtime/lib/mixins/-proxy.js

```diff
@@ -83,9 +79,11 @@ export default Mixin.create({
   */
   content: null,
 
-  _initializeTag: on('init', function() {
+  init() {
+    this._super(...arguments);
+    meta(this).setProxy();
```

Comment:
Hmm. Actually, I just noticed that we still need to fix the issue that @chancancode mentioned below.

The tag needs to be set on meta in an on('init').